config/environments/production.rb

```diff
@@ -29,4 +97,3 @@
     "Cache-Control" => "public, max-age=31557600",
   }
 end
-Rack::Timeout.timeout = ENV.fetch("RACK_TIMEOUT", 10).to_i
```

Comment:
This removal is on purpose - the updated version (0.5.1) of rack-timeout [behaves this way by default](https://github.com/heroku/rack-timeout#rails-apps) and doesn't need the setting in place.